src/commands/deploy/command.ts

```diff
@@ -83,5 +89,9 @@ function decideDeployType(): DeployType {
         }
     }
 
+    if (fs.existsSync("Dockerfile")) {
```

Comment:
this means the CLI will assume a docker deployment whenever it detects a Dockerfile? if I would like to switch to classic deployment I should just delete de Dockerfile?

Reply:
Yea, or move the Dockerfile or rename it... it is not ideal, but we wanted to get this started.